spec/jobs/google/search_keyword_job_spec.rb

```diff
@@ -133,9 +136,19 @@
         allow(described_class).to receive(:perform_now)
 
         described_class.perform_now keyword_id
+      rescue Google::Errors::SearchKeywordError
 
         expect(described_class).to have_received(:perform_now).with(keyword_id).exactly(:once)
       end
+
+      it 'raises SearchKeywordError' do
```

Comment:
```suggestion
      it 'raises SearchKeywordError to allow Sidekiq to retry the search' do
```

Should we document this implicit important concept? :)